templates/electron-app/webpack.config.babel.js

```diff
@@ -0,0 +1,120 @@
+import path from 'path';
+import webpack from 'webpack';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+
+const env = process.env.NODE_ENV || 'production';
+
+let plugins = [
+  new CopyWebpackPlugin([{ from: './public' }]),
+  new CopyWebpackPlugin([{ from: './electron-entry.js' }, { from: './package.json' }]),
```

Comment:
Line 9 was added to copy the electron entrypoint script and package.json into dist